Comparison Engine/Comparison Engine/Form Classes/ListButtonMethods.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Comparison_Engine.Base_Classes;
+using System.Collections.Generic;
+
+namespace Comparison_Engine.Form_Classes
+{
+    class ListButtonMethods
```

Comment:
Just for clarity's sake, would be nice to have more distinct names for "ButtonListMethods.cs" and "ListButtonMethods.cs" since they are so similar

Reply:
Renamed "ListButtonMethods.cs" to "ListMethods.cs"